### DISCUSSION TITLE
How to set the database task automatic execution of Firebird Database?

### DISCUSSION THREAD

**Discussion by bgmxl** [dbeaver/dbeaver#18356] [Q&A]
I have a period of SQL and wants it to execute it once a day, how to set it.
![image](https://user-images.githubusercontent.com/34325905/204261877-5f9f3b38-9ccf-44ec-a0e3-54831b44844c.png)

**Comment by LonwoLonwo** [comment]
Hello @bgmxl 

You can read more about task scheduling here: https://github.com/dbeaver/dbeaver/wiki/Task-Scheduler.

You can schedule your task only in PRO DBeaver's versions. Or you can investigate your system scheduler as well.

**Reply by bgmxl** [reply]
thank you very much.
